modules/customizing-the-web-console-URL.adoc

```diff
@@ -10,12 +10,12 @@ For `console` and `downloads` routes, custom routes functionality uses the `ingr
 [id="customizing-the-console-route_{context}"]
 == Customizing the console route
```

Comment:
I couldn't find this in the preview. Could you link me?

Reply:
@EricPonvelle Sorry, I tossed these updates in last minute. Preview is here: https://deploy-preview-34947--osdocs.netlify.app/openshift-enterprise/latest/web_console/customizing-the-web-console.html#customizing-the-web-console-url_customizing-web-console